src/main/java/com/microsoft/azure/datalake/store/HttpTransport.java

```diff
@@ -32,12 +32,19 @@
  *     makeCall - wraps retries around makeSingleCall
  * </P>
  */
-class HttpTransport {
+public class HttpTransport {
 
     private static final String API_VERSION = "2017-08-01"; // API version used in REST requests
     private static final Logger log = LoggerFactory.getLogger("com.microsoft.azure.datalake.store.HttpTransport");
     private static final Logger tokenlog = LoggerFactory.getLogger("com.microsoft.azure.datalake.store.HttpTransport.tokens");
 
+    private static Proxy proxy;
```

Comment:
We could initialize this to `Proxy.NO_PROXY` and then enforce non-null in the setter, which seems a bit cleaner and simplifies the open call.